Tasks/XcodeV5/xcode.ts

```diff
@@ -21,7 +21,10 @@ async function run() {
         if (xcodeVersionSelection === 'specifyPath') {
             let devDir = tl.getInput('xcodeDeveloperDir', true);
             tl.setVariable('DEVELOPER_DIR', devDir);
-            telemetryData.xcodeDeveloperDir = devDir;
+            const xcodeFileName: string = devDir.split('/').find(item => item.includes('.app'));
```

Comment:
Can `devDir` be undefined?

Reply:
I believe yes, if customer' dir will be not contain `.app` part.

Reply:
I asked about `devDir`, not `xcodeFileName`.
This line could fail `devDir.split('/')` if `devDir` is undefined so would like to confirm that it will be always defined

Reply:
Fixed